auditbeat/module/audit/file/event.go

```diff
@@ -211,7 +180,7 @@ func (e *Event) String() string {
 	return string(data)
 }
 
-func buildMapStr(e *Event) common.MapStr {
+func buildMapStr(e *Event, existsBefore, existsNow bool) common.MapStr {
```

Comment:
Could existsNow be determined by looking at e? Then you wouldn’t need the third param.